packages/flame/lib/src/components/route.dart

```diff
@@ -36,6 +37,13 @@ class Route extends PositionComponent with ParentIsA<RouterComponent> {
   /// route underneath doesn't need to be rendered.
   final bool transparent;
 
+  /// If false, the route will not maintain the state of this route's page
+  /// component.  By default, once a route becomes active, the component
+  /// built by the build routine is maintained by the route after the route
+  /// is popped off the stack.  Setting [pageTransient] to false will drop the
+  /// page component when the route is popped off the stack.
+  final bool pageTransient;
```

Comment:
I'm reading this description, and just can't wrap my head around it.

"Transient" means temporary, short-lived, impermanent. So, if the page is transient (`pageTransient` is true), that has to mean that the page will be short-lived, that it will be deleted once the route is popped.

However, this is not the default behavior. The default behavior of a route is that a page is persistent: once built, it will remain attached to the route and will not need to be recreated in the future. Thus, the default behavior is that a page is persistent, that is `pageTransient = false`.

It appears then that the property is currently doing the opposite of what it's supposed to be doing.

Reply:
You're right - I don't know how I flipped that in my head.  Let me redo this to make more sense and fix the wording.

Reply:
Maybe it could be flipped around, and be called `pagePermanent` or something like that instead? I think that would also be clearer to the users what it does

Reply:
@pastordougdev what do you think about flipping it around?